src/PHPMailer.php

```diff
@@ -148,6 +157,13 @@ class PHPMailer
      */
     public $Ical = '';
 
+    /**
+     * Value of "method" in Contenttype header "text/calendar"
+     *
+     * @var string
+     */
+    public $IcalMethod = self::ICAL_METHOD_REQUEST;
```

Comment:
I think it would be safer and simpler to skip this property and extract the method from the `Ical` property instead, using the permitted values declared in the constants. Otherwise you're leaving it open to the user setting a method that does not match the content.

Reply:
Also all references to these constants should be `static` rather than `self` to allow overriding.

Reply:
If you say skippen, do you mean remove it from the Header or ignoring it in processing the icalendar?